DllImportGenerator/Demo/Program.cs

```diff
@@ -31,13 +31,6 @@ static void Main(string[] args)
             c = b;
             NativeExportsNE.Sum(a, ref c);
             Console.WriteLine($"{a} + {b} = {c}");
-
-            SafeHandleTests tests = new SafeHandleTests();
```

Comment:
Should the project stop referencing `xunit` now?

Reply:
Yes.